### PR TITLE
SE-843-modify-deployment-abi-loader

### PR DESCRIPTION
**Pull request by ShaneDuncan602**

### CODE REVIEW

File: deployments/1.3.1/abi/diamonds.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{
+    "1.3.1": {
+        "ProtocolProcessorDiamond": {
+            "VersionFacet": "0xf6b3B1FD3a75DE3cB7d287b5a2847F327C09865f",
+            "ProtocolNativeFacet": "0xc4cb6f7798DfE920AC83CE3D9867046b95FF75af",
+            "ProtocolRawFacet": "0xa647b1D689e97F6cf9A34eC7eF24B0E87917745b",
+            "ERC20RuleProcessorFacet": "0x4096989b910d6dF88b1c0FD16421cB70fA88eae0",
+            "ERC721RuleProcessorFacet": "0x7179177bDE758Bed7344Bc7fA70e85FC1B26b097",
+            "diamond": "0x5C86D6742A1D0c12cd349518549ff8992838f50F",
```
Comment (VoR0220) [inline deployments/1.3.1/abi/diamonds.json]: Maybe we capitalize this?

**Comment by VoR0220** [comment]
Are we planning to add deployments into the gitignore?

**Review by TJ-Everett — APPROVED** [review]
Approved for me, looks good!

File: upgrade_version.sh [hunk 2]
```diff
@@ -71,7 +78,10 @@ main() {
     # replace in package.json
     replace_version_in_package_json "./package.json"  "$new_version"
 
-    # replace in package.json
+    # replace in script
+    replace_version_in_abi_script "./script/python/record_abi.py" "$new_version"
```
Comment (VoR0220) [inline upgrade_version.sh:82]: Does this also do it for record_facets.py as well?

**Comment by ShaneDuncan602** [comment]
> Are we planning to add deployments into the gitignore?

No. The deployments will be pushed to main so DOOM can use them.

**Review by VoR0220 — APPROVED** [review]

**Pull request merged**